recipes/uppasd/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "uppasd" %}
+{% set version = "v6.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UppASD/UppASD/releases/download/{{ version }}/{{ version }}.tar.gz
+  sha256: f483bb6e0a5c75820e4a3d38894c40557548777160eabd3c152c8c356e6ca8ed
+  patches:
+    - binary-name.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}  # [not win]
+    # Windows: complation with flang fails because the provided flang version 5 does not provide
+    # a NORM2 intrinsic; wait for https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/5081
+    - {{ compiler('m2w64_fortran') }}  # [win]
```

Comment:
The `flang19` migration is ongoing, perhaps we can skip the Windows build for now and enable it again after the PR has been merged? fyi @conda-forge/flang

Reply:
What is the timeline for that migration? We have an ongoing project where we would want to depend on this package *on Windows* by the end of the year.

Reply:
The migration  as an whole is taking some time, but that does not prevent you to opt in it. For example, we migrated to the new flang 19 compiled for mumps back in april: https://github.com/conda-forge/mumps-feedstock/pull/134 and it has been working fine.

Reply:
@h-vetinari I have changed this as suggested but the build on Windows is now failing because it [cannot find `omp_lib`](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1390950&view=logs&j=53c3d5f4-e7c7-5b53-4b94-211701b2ba17&t=8f042650-8479-5761-7a4a-8d021c0bb2de&l=2501). What's the correct way to get OpenMP with flang19.

Reply:
For omp_lib.mod (a precompiled fortran module), you have to use a separate output, i.e. add `- llvm-openmp-fortran  # [win]` to your host section.

For the migrator you shouldn't have to worry, it'll come by by itself. But all the magic boils down to copying the `flang19.yaml` file from the global pinning to the `.ci_support/migrations` folder on the feedstock and then rerendering.